ci/tasks/light-aws/run-upload-test.sh

```
@@ -0,0 +1,18 @@
+REPO_PARENT="$( cd "${REPO_ROOT}/.." && pwd )"
+
+if [[ -n "${DEBUG:-}" ]]; then
+  set -x
+  export BOSH_LOG_LEVEL=debug
+  export BOSH_LOG_PATH="${BOSH_LOG_PATH:-${REPO_PARENT}/bosh-debug.log}"
+fi
+
+source "${REPO_PARENT}/director-state/director.env"
+
+pushd "${REPO_PARENT}/stemcell"
+  time bosh -n upload-stemcell *.tgz
```

Shellcheck (0.11.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)